operator/helper/multiline_test.go

```diff
@@ -164,8 +160,39 @@ func TestLineStartSplitFunc(t *testing.T) {
 				"LOGPART log1\nLOGPART log1",
 			},
 			Flusher: &Flusher{
-				force:           true,
-				lastForcedFlush: time.Now(),
+				// We assume than in previous iteration we had same data length
```

Comment:
I can't figure out why this is necessary.

Reply:
According to the explanation with list. This is `2b` case, so we have nth iteration of loop without any flush and new data added.

In other words, this unit case covers force flushing